subsys/net/lib/tftp/Kconfig

```diff
@@ -0,0 +1,64 @@
+# Socket MQTT Library for Zephyr
+
+# Copyright (c) 2018 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config TFTP_LIB
+	bool "Socket TFTP Library Support"
+	select NET_SOCKETS
+	select NET_SOCKETS_POSIX_NAMES
+	help
+	  Enable the Zephyr TFTP Library
+
+if TFTP_LIB
+
+module=TFTP
+module-dep=NET_LOG
+module-str=Log level for TFTP
+module-help=Enables tftp debug messages.
+source "subsys/net/Kconfig.template.log_config.net"
+
+config TFTPC_DEFAULT_SERVER_IP
+	string "Default IP address that the TFTP Client will try to connect to."
+	default "192.168.0.100"
```

Comment:
I think there should not be a default value for IP address. This value is only useful in your own environment.